css-device-adapt/Overview.bs

```diff
@@ -66,6 +62,65 @@ for a good discussion of these issues.
 Issue: Various issues about this specification and related specifications
 are listed in <a href="https://www.w3.org/Graphics/SVG/WG/wiki/Proposals/Investigation_of_APIs_for_Level_of_detail#The_issues_on_existing_APIs">this report</a>.
 
+<h2 id="scenarios">
+Motivating Scenarios</h2>
+
+<div class="example">
+    In this example, the document can be rendered without issue with any
+    size viewport. The author indicates this using the
+    <code class=css>@viewport</code> rule.
+
+    <pre class="lang-html">
+        &lt;!doctype html>
+        &lt;html>
+        &lt;head>
+        &lt;title>My Site&lt;/title>
+        &lt;style>
+            @viewport { width: device-width; }
```

Comment:
should be `@viewport { width: auto; }`. `width: device-width` was removed from the spec quite a while back.